From b0246f357b80f91811a3eb80f9cf309b54ef8f15 Mon Sep 17 00:00:00 2001
From: Julian Ospald <hasufell@posteo.de>
Date: Fri, 23 Mar 2018 17:01:02 +0100
Subject: [PATCH] Use mpv instead of mplayer
Upstream: no

---
 impressive.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/impressive.py b/impressive.py
index 1dbbcfa..8951aa0 100755
--- a/impressive.py
+++ b/impressive.py
@@ -179,7 +179,7 @@ if os.name == "nt":
 else:
     def FindBinary(x): return x
     GhostScriptPlatformOptions = []
-    MPlayerPath = "mplayer"
+    MPlayerPath = "mpv"
     pdftkPath = "pdftk"
     mutoolPath = "mutool"
     ffmpegPath = "ffmpeg"
-- 
2.16.2

