From c8e5c7e082b3c0a0d4da0a7f299ddb71653fd268 Mon Sep 17 00:00:00 2001
From: Julian Ospald <hasufell@posteo.de>
Date: Fri, 26 May 2017 16:08:12 +0200
Subject: [PATCH] Fix build with OpenSceneGraph-3.5.6

---
 components/resource/bulletshapemanager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/resource/bulletshapemanager.cpp b/components/resource/bulletshapemanager.cpp
index 0a5ed78..2e18dc8 100644
--- a/components/resource/bulletshapemanager.cpp
+++ b/components/resource/bulletshapemanager.cpp
@@ -42,7 +42,7 @@ struct GetTriangleFunctor
         return btVector3(vec.x(), vec.y(), vec.z());
     }
 
-    void inline operator()( const osg::Vec3 v1, const osg::Vec3 v2, const osg::Vec3 v3, bool _temp )
+    void inline operator()( const osg::Vec3 v1, const osg::Vec3 v2, const osg::Vec3 v3 )
     {
         if (mTriMesh)
             mTriMesh->addTriangle( toBullet(mMatrix.preMult(v1)), toBullet(mMatrix.preMult(v2)), toBullet(mMatrix.preMult(v3)));
-- 
2.13.0

