From e28f40d13da702dd5074b85996ea520006f6d3fb Mon Sep 17 00:00:00 2001
From: Arnaud Rebillout <elboulangero@gmail.com>
Date: Sat, 27 May 2017 14:51:17 +0700
Subject: [PATCH] Fix manpage destination

---
 data/man/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/man/CMakeLists.txt b/data/man/CMakeLists.txt
index 471ec9a..f7db0ba 100644
--- a/data/man/CMakeLists.txt
+++ b/data/man/CMakeLists.txt
@@ -1,5 +1,5 @@
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/pnmixer.1.in
    ${CMAKE_CURRENT_BINARY_DIR}/pnmixer.1)
 
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/pnmixer.1" DESTINATION "${CMAKE_INSTALL_MANDIR}")
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/pnmixer.1" DESTINATION "${CMAKE_INSTALL_MANDIR}/man1")
 
